cpu/driver/runtest.sh

```diff
@@ -34,6 +53,18 @@ function verify_intel_cpufreq_driver
     typeset vendor=$(dmidecode -t 0 | grep Vendor: | \
                      cut -d: -f 2 | awk '{print tolower($1)}')
 
+    # make sure the given system supports p-state
+    check_pstate_support
+    if [ $? -ne 0 ]; then
+	# older systems do not support intel pstate
+	if [ $driver != "acpi-cpufreq" ]; then
+	    cki_log "intel (non-pstate) system is running: $driver"
+	    return $CKI_FAIL
```

Comment:
should we skip instead ?

Reply:
I we'd typically expect some cpu frequency driver to be running.  I guess they are all old systems so perhaps SKIP is better.